src/community/index.html

```diff
@@ -0,0 +1,115 @@
+    <main x-data="state" class="flex items-center flex-col grow sm:grow-0 width-[448px] sm:w-[780px] min-w-0">
+      <header class="min-w-auto sm:min-w-[640px] sm:max-w-[640px] mb-8">
+        <a href="/" class="min-h-auto pt-2 sm:pt-0 sm:min-h-[546px] flex flex-col items-center mb-2">
+          <div class="flex sm:hidden p-5 rounded-full border-2 border-bright_black justify-center items-center">
+            <img alt="srcery glyph" src="../svg/glyph.svg" class="glyph w-logo h-logo">
+          </div>
+          <div class="bg-triangle bg-no-repeat min-h-[546px] pt-[30%] hidden sm:flex min-w-full bg-center align-center justify-center">
+            <img alt="srcery glyph" src="../svg/glyph.svg" class="glyph w-logo h-logo">
+          </div>
+          <div class="mt-4 sm:mt-[-18%] flex items-center">
+            <div class="flex mr-2">
+              <div class="dot bg-xgray2"></div>
+              <div class="dot bg-xgray7"></div>
+              <div class="dot bg-xgray10"></div>
+            </div>
+            <h1 class="uppercase text-6xl">srcery</h1>
+            <div class="flex ml-2">
+              <div class="dot bg-xgray10"></div>
+              <div class="dot bg-xgray7"></div>
+              <div class="dot bg-xgray2"></div>
+            </div>
+          </div>
+        </a>
+        <nav class="flex justify-center max-w-[384px] sm:max-w-full flex grow justify-between">
+          <span class="w-full text-left">
+            <a href="/themes/" class="link uppercase">themes</a>
+          </span>
+          <span class="w-full text-center">
+            <a href="/resources/" class="link uppercase">resources</a>
+          </span>
+          <span class="w-full text-right">
+            <a href="/community/" class="link uppercase text-bright_blue border-bright_blue">community</a>
+          </span>
+        </nav>
+      </header>
+      <section class="w-full">
+        <div class="mb-8">
+          <h2 class="text-xl mb-3 text-center sm:text-left uppercase">
+            <!-- <div class="svg w-8 mr-2" x-html="icon"></div> -->
+            Discord
+          </h2>
+          <p class="text-justify mb-3">
+            We run a discord server for those interested, join us there if you
+            have any burning questions or just wanna say hi.
+          </p>
+          <a href="https://discord.gg/G6vBMmZ" target="_blank" class="flex justify-center sm:justify-start">
+            <img src="https://img.shields.io/discord/714101903377694741?color=%232C78BF&label=discord&logo=discord">
+          </a>
+        </div>
+
+        <div class="mb-8">
+          <h2 class="text-xl mb-3 text-center sm:text-left uppercase">Merchandise</h2>
+          <p class="text-justify mb-3">
+            Get our brand new merchandise featuring the Srcery color scheme!
+            We're not making any profit from the sales, so you can show your
+            support at cost. Thank you for helping us develop and improve our
+            color scheme.
```

Comment:
> so you can show your support at cost.

What do you mean?

Reply:
I meant that we're not making any profit, the way you show support is by using srcery merch. Could possibly be rephrased, but blame chatGPT, got the bot to output some copy :D

Reply:
My bad, missed another commit. Fixed the copy now, with the latest revisions